post-processing/fuzz_data_loader.py

```diff
@@ -260,6 +260,8 @@ def __init__(self, profiles: List[FuzzerProfile]):
             incoming_references = list()
 
             for reached_func_name in fp_obj.functions_reached:
+                if reached_func_name not in self.all_functions:
+                    continue
```

Comment:
Could you add an error logging here? 
`l.error("Found mismatched function name between all_functions and functions_reached")`